src/intl/resolver.js

```diff
@@ -257,6 +257,9 @@ function* toString(entity) {
 }
 
 export function format(ctx, args, entity) {
+  if (entity === undefined) {
+    return ['', []];
+  }
```

Comment:
I've seen this too.  I'll investigate and maybe fix on master.  This looks like a right fix.